docs/docs/cmd/pp/chatbot/chatbot-get.md

```diff
@@ -0,0 +1,123 @@
+# pp chatbot get
+
+Gets a specific Microsoft Power Platform chatbot in the specified Power Platform environment
+
+## Usage
+
+```sh
+pp chatbot get [options]
+```
+
+## Options
+
+`-e, --environment <environment>`
+: The name of the environment.
+
+`-i, --id [id]`
+: The id of the chatbot. Specify either `id` or `name` but not both.
+
+`-n, --name [name]`
+: The name of the chatbot. Specify either `id` or `name` but not both.
+
+`-a, --asAdmin`
```

Comment:
Let's remove the short option here. For reference, you can take a peek at issue #4132. 😄